hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/IndexListenerService.kt

```diff
@@ -114,6 +114,28 @@ class IndexListenerService(
         }
       }
 
+  private fun reindexPrisonerOld(ob: OffenderBooking, eventType: String): Prisoner? =
+    indexStatusService.getIndexStatus()
+      .run {
+        if (activeIndexesEmpty()) {
+          log.info("Ignoring update (old) of prisoner {} as no indexes were active", ob.offenderNo)
+          null
+        } else {
+          val prisoner = prisonerSynchroniserService.reindex(ob, activeIndexes(), eventType)
+          prisoner
```

Comment:
No val is needed here

Reply:
There's another one on line 111 which might be clearer as:
```
prisonerSynchroniserService.reindex(ob, activeIndexes(), eventType)
  .also { prisonerSynchroniserService.reindexUpdate(ob, eventType) }
```

Reply:
I could just reorder them